src/index.tsx

```diff
@@ -50,8 +49,8 @@ ReactDOM.render(
                 <Route path='/get-access' exact component={Invite} />
                 <Route path='/login' exact component={LoginPage} />
                 <Route path='/reset-password' exact component={ResetPassword} />
-                <Route path='/:znsRoute?/' exact render={redirectToDefaults} />
-                <Route path='/:znsRoute/:app' component={ZnsRouteConnect} />
+                <Route path='/' exact component={MessengerMain} />
+                <Route path='/:znsRoute' render={redirectToDefaults} />
```

Comment:
Is the `ZnsRouteConnect` component used any more?

Reply:
Not used anymore, we don't need the complexity for the setting routes. The component and test file now removed :) If all looks good I'll merge this one in.